distribution/lib/Standard/Base/0.0.0-dev/src/Data/Number_Builtins.enso

```diff
@@ -0,0 +1,31 @@
+private
+
+integer_plus a b = @Builtin_Method "Integer.+"
```

Comment:
@GregoryTravis, @radeusgd, @jdunkerley, @AdRiley, this would be my preferred way to handle `# PRIVATE` properly. Move all the _builtins calls_ into a separate file with `private` on the top. That makes the functions in this file inaccessible from outside of the project. One can still call them from `Numbers.enso` & co. however.

Reply:
Honestly, I miss the `.h` and `.c` files separation. As such I prefer putting `private` stuff in its own file, rather than marking stuff `private` in the same file. In the terms of [clarity of access modifiers](http://wiki.apidesign.org/wiki/ClarityOfAccessModifiers) and [clarity of types](http://wiki.apidesign.org/wiki/ClarityOfTypes) - I prefer types - e.g. Enso modules.